BUILD.boost

```diff
@@ -800,8 +803,6 @@ boost_library(
     hdrs = [
         "boost/cstdint.hpp",
         "boost/integer_traits.hpp",
-        "boost/pending/integer_log2.hpp",
```

Comment:
`boost/pending/integer_log2.hpp` is actually still in boost-1.71.0, but is deprecated and a forward to `boost/integer/interger_log2.hpp`. We could keep it here if we wanted to.

`boost/pending/lowest_bit.hpp` is altogether missing.